src/python/engagement_edge/src/engagement_edge.py

```diff
@@ -444,23 +187,7 @@ def user_auth_table():
     return DYNAMO.Table(os.environ["USER_AUTH_TABLE"])
 
 
-def create_user(username, cleartext):
```

Comment:
removing this, in particular, freaked me out - but searching by the pepper, it looks like the only palces we create users now are
- grapl provision (the service)
- grapl provision (the ipynb)